tests_pytest/common_tests/unit_tests/test_base_graph.py

```diff
@@ -0,0 +1,215 @@
+                                                    is_activation_quantization_enabled=True,
+                                                    candidates_quantization_cfg=copy.deepcopy(candidate_single)))
+        mock_nodes_list.append(create_mock_base_node(name='conv_2', layer_class='Conv2d', 
+                                                    is_weights_quantization_enabled=True, is_fln_quantization=True,
+                                                    candidates_quantization_cfg=copy.deepcopy(candidate_single)))
+        mock_nodes_list.append(create_mock_base_node(name='tanh', layer_class='Tanh', 
+                                                    is_activation_quantization_enabled=True,
+                                                    candidates_quantization_cfg=copy.deepcopy(candidate_single)))
+        mock_nodes_list.append(create_mock_base_node(name='conv_3', layer_class='Conv2d', 
+                                                    is_weights_quantization_enabled=True, is_fln_quantization=True,
+                                                    candidates_quantization_cfg=copy.deepcopy(candidates_multiple)))
+        mock_nodes_list.append(create_mock_base_node(name='relu_2', layer_class='ReLU', 
+                                                    is_activation_quantization_enabled=True,
+                                                    candidates_quantization_cfg=copy.deepcopy(candidates_multiple)))
+        mock_nodes_list.append(create_mock_base_node(name='flatten', layer_class='Flatten', 
+                                                    is_quantization_preserving=True,
+                                                    candidates_quantization_cfg=copy.deepcopy(candidate_single)))
+        mock_nodes_list.append(create_mock_base_node(name='linear', layer_class='Linear', 
+                                                    is_weights_quantization_enabled=True, is_activation_quantization_enabled=True,
+                                                    candidates_quantization_cfg=copy.deepcopy(candidates_multiple)))
+        self.mock_node = mock_nodes_list
+
+        edges_list = []
+        for i in range(1, len(mock_nodes_list), 1):
+            edges_list.append(Edge(mock_nodes_list[i-1], mock_nodes_list[i], 0, 0))
+
+        # Create a mock graph
+        self.graph = Graph('g',
+                            input_nodes=[mock_nodes_list[0]],
+                            nodes=mock_nodes_list,
+                            output_nodes=[OutTensor(mock_nodes_list[5], 0)],
+                            edge_list=edges_list)
+
+    @pytest.fixture(autouse=True)
+    def expected_candidates_fixture(self):
+        """
+        Set up and create expected values and candidates for tests related to the disable_fused_nodes_activation_quantization modules.
+        """
+        def create_exp_candidate_cfg(candidate, quant_mode, n_bits, actq_params_fn, actq_method, signedness=None):
```

Comment:
I suggest you split this setup function, extract `create_exp_candidate_cfg` to an outer scope (no need for nested function here) and maybe create a function that gets the arguments to build and set a config, instead of working with these long dictionaries.
All this is just for simplification and readability.